README.md

```diff
@@ -371,6 +371,46 @@ in data.py.
         name: util-linux efitools
 ```
 
+## install_xcpng.py
+
+This script install (or upgrade) a VM into a XCP-ng host using a PXE server. You need to have the PXE server configured and running with the proper installer for each version. For more informations about PXE boot install you can read the XCP-ng documentation [here](https://xcp-ng.org/docs/install.html#pxe-boot-install).
+
+The script can display the help with `-h`:
+
+```
+usage: install_xcpng.py [-h] [--installer INSTALLER] host vm_uuid action version_xcpng
+
+positional arguments:
+  host                  The IP address or the name of the host where the VM is created.
+  vm_uuid               The UUID of an existing VM to use to create the new host.
+  action                install or upgrade.
+  version_xcpng         The version of xcp-ng we wish.
+
+options:
+  -h, --help            show this help message and exit
+  --installer INSTALLER
+                        URL of the installer on the PXE
+```
+
+Here an example of the command for an installation:
+```
+python scripts/test_install_xcpng.py 10.0.0.2 3bdf2cc6-4e6e-526d-4f18-bf7899953af6 install 8.2.1
+```
+
+Here another example for an upgrade:
+```
+python scripts/test_install_xcpng.py 10.0.0.2 f0f5f010-80c6-25ae-44a2-1fb154e32d14 upgrade 8.2.1
+```
+
+The script use a variable from the `data.py` file. You need to add the following parameter inside that file.
+```
+# Default PXE server
+DEF_PXE_IP = '10.0.0.2'
+```
+
+The `installer` parameter isn't mandatory. If you leav it empty it will be created like this: http://<PXE_IP>/installers/xcp-ng/<version>/
```

Comment:
```suggestion
The `installer` parameter isn't mandatory. If you leave it empty it will be created like this: http://<PXE_IP>/installers/xcp-ng/<version>/
```